new-pipeline/src/dashboards.js

```diff
@@ -1046,16 +1046,52 @@ function updateOSs() {
   });
 }
 
+// Build a URL for linking to the probe-dictionary.
+function buildDictionaryURL(metric, channel, description) {
+  // The probe-dictionary doesn't know about simple measures yet.
+  if (metric.startsWith("SIMPLE_MEASURES_")) {
+    return null;
+  }
+
+  var baseUrl = "https://georgf.github.io/fx-data-explorer/index.html";
+  var params = {
+    "searchtype": "in_name",
+    "optout": "false",
+    "channel": channel,
+    "constraint": "is_in",
+    "version": "any",
+  };
+
+  if (metric.startsWith("SCALARS_")) {
+    // Scalar naming in the aggregates is different from the Firefox names.
+    metric = metric.replace(/^SCALARS_/, "").toLowerCase();
+    params["detailView"] = "scalar/" + metric;
+  } else {
+    // All other probes in the aggregates should be histograms.
+    params["detailView"] = "histogram/" + metric;
+  }
+
+  params["search"] = metric.toLowerCase();
+  return baseUrl + "?" + $.param(params);
+}
+
 function getDescriptionWithLink(metric, channel, description) {
-  var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
-  var descr = description === null ? metric : description
+  var metricUrl = buildDictionaryURL(metric, channel, description);
+  var descr = metric;
+  if (description && (description.length > 0)) {
+    descr = description;
+  }
 
   var div = document.createElement('div');
   div.classList.add("text-center");
-  div.innerHTML =  descr + " <a href=\"" + metricUrl + "\" target=\"_blank\">" +
-                   "<i class=\"btn btn-outline-primary fa fa-info-circle\" " +
-                   "style='color: black' aria-hidden=\"true\">  " +
-                   "More Details </i></a>"
+  var html = descr;
```

Comment:
Using `innerHTML`, you should wrap this in a tag. Or we should build the nodes using DOM APIs since metricURL contains user-supplied input, then we can avoid `innerHTML` entirely.